DeepCat.js

```diff
@@ -8,7 +8,8 @@
 ( function( $, mw ) {
 	var DeepCat = {},
 		keyString = 'deepcat:',
-		maxDepth = 15,
+		defaultDepth = 15,
+		maxDepth = 20,
```

Comment:
thiemowmde: CatGraph itself will happily recurse into all categories of a toplevel category of any wiki in a couple seconds. A limit of 500 result rows (not recursion depth) is configured in catgraph-jsonp, it will never return more than that. An additional recursion limit in DeepCat doesn't hurt, I think.